packages/patternfly-3/patternfly-react/src/components/LoadingState/LoadingState.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from '../Spinner';
+
+class LoadingState extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      render: false
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
```

Comment:
Need to set this into a property and clear it if the component unmounts before the function is fired.